framework/src/source/ConsoleTestReporter.bs

```diff
@@ -13,7 +13,7 @@ namespace rooibos
       end if
     end function
 
-    override function reportResults(allStats)
+    override function onEnd(allStats)
       m.allStats = allStats
```

Comment:
```suggestion
    override function onEnd(event as object)
      m.allStats = event.stats
```

Reply:
Done. Unfortunately I couldn't do an interface in this one like I did for `onEnd`, since brighterscript doesn't like it when interfaces are empty.